cunumeric/random/random.py

```diff
@@ -28,7 +28,14 @@
     from ..types import NdShapeLike
 
 
+default_rng = generator.default_rng
+
+
 def seed(init: Union[int, None] = None) -> None:
+    """
```

Comment:
Should this function have an "Availability" section?

Reply:
This function is currently a no-op when cuRAND is used. Will fix it in a follow-up PR and also add an Availability section then.